main/app.go

```diff
@@ -18,31 +21,46 @@ type App struct {
 	DB     *sql.DB
 }
 
-//Initialize method
-func (a *App) Initialize(user, password, dbname string) {
-	connectionString := fmt.Sprintf("user=%s password=%s dbname=%s sslmode=disable", user, password, dbname)
+//Initialize func: init server according configuration structure
+func (a *App) Initialize(config Configuration) {
 	var err error
-	a.DB, err = sql.Open("postgres", connectionString)
-	if err != nil {
-		log.Fatal(err)
+	if !config.Debug {
+		var PgDbURL = config.PgDbURL
+		dbURLSliced := strings.Split(PgDbURL, ":")
+		port, err := strconv.Atoi(dbURLSliced[1])
+		if err != nil {
+			log.Fatal(err)
+		}
+		connectionString := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable",
+			dbURLSliced[0], port, config.PgUsername, config.PgPassword, config.PgDatabase)
+		a.DB, err = sql.Open("postgres", connectionString)
+		if err != nil {
+			log.Fatal(err)
+		}
+	} else {
+		a.DB, err = sql.Open("sqlite3", "entities.db")
+		if err != nil {
+			log.Fatal(err)
+		}
+		CreateTable(a.DB)
 	}
 	a.Router = mux.NewRouter()
 	a.InitializeRoutes()
 }
 
-//Run method
+//Run server
 func (a *App) Run(addr string) {
 	log.Fatal(http.ListenAndServe(addr, a.Router))
 }
 
-//InitializeRoutes method
+//InitializeRoutes - init routes for api requests
 func (a *App) InitializeRoutes() {
 	a.Router.HandleFunc("/", a.Ok).Methods("GET")
 	a.Router.HandleFunc("/entities", a.GetEntities).Methods("GET")
 	a.Router.HandleFunc("/entity", a.CreateEntity).Methods("POST")
-	a.Router.HandleFunc("/entity/{id:[0-9]+}", a.GetEntity).Methods("GET")
-	a.Router.HandleFunc("/entity/{id:[0-9]+}", a.UpdateEntity).Methods("PUT")
-	a.Router.HandleFunc("/entity/{id:[0-9]+}", a.DeleteEntity).Methods("DELETE")
+	a.Router.HandleFunc("/entity/{id:[a-z0-9_-]*}", a.GetEntity).Methods("GET")
```

Comment:
Too broad: UUID v4 contains only hex numbers and `-`, also, it has fixed size, 36 chars always